src/opFn.ts

```diff
@@ -0,0 +1,220 @@
+import { State } from 'src/State';
+import { VmStop } from 'src/exceptions';
+import { U256 } from 'src/U256';
+
+export const opTemp = (state: State) => {
+
+}
+
+export const opStop = (state: State) => {
+	// TODO
+	throw new VmStop();

```

Comment:
exception for VmStop?

Reply:
I want to replace it on state.eei.finish()
in next pr